src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -53,7 +53,6 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
 
     // type name for this object, used as the ES-type
     const esType = config.type;
-    this.index = kbnIndex;
```

Comment:
I couldn't find any instance of it being used and never ran into an error when testing.